execute-generators/README.md

```diff
@@ -27,45 +24,45 @@ required arguments:
-  --plugin PLUGIN                         plugin to to load. The format is
-                                          --plugin=<id>::<path>
+  --plugin PLUGIN                         plugin to to load. The format is --plugin=<id>::<path>
         
-  --macro MACRO                           macro to define. The format is
-                                          --macro=<name>::<value>
+  --macro MACRO                           macro to define. The format is --macro=<name>::<value>
         
   --plugin-location PLUGIN_LOCATION       location to load additional plugins from
         
-  --build-number BUILD_NUMBER             build number used to determine if the
-                                          plugins are compatible
+  --build-number BUILD_NUMBER             build number used to determine if the plugins are
+                                          compatible
         
   --test-mode                             run in test mode
         
-  --environment ENVIRONMENT               kind of environment to initialize,
-                                          supported values are 'idea' (default),
-                                          'mps'
-                                      
-  --parallel-generation-threads THREADS   number of threads to use for parallel
-                                          generation. Default: 0 (parallel
-                                          generation disabled).                                      
-
-  --log-level LOG_LEVEL                   console log level. Supported values:
-                                          info, warn, error, off. Default: warn.
+  --environment ENVIRONMENT               kind of environment to initialize, supported values are
+                                          'idea' (default), 'mps'
+
+  --log-level LOG_LEVEL                   console log level. Supported values: info, warn, error,
+                                          off. Default: warn.
         
   --model MODEL                           list of models to generate
         
   --module MODULE                         list of modules to generate
         
-  --exclude-model EXCLUDE_MODEL           list of models to exclude from
-                                          generation
+  --exclude-model EXCLUDE_MODEL           list of models to exclude from generation
         
-  --exclude-module EXCLUDE_MODULE         list of modules to exclude from
-                                          generation
+  --exclude-module EXCLUDE_MODULE         list of modules to exclude from generation
+
+  --parallel-generation-threads THREADS   Number of threads to use for parallel generation. Value
+                                          of 0 means that parallel generation is disabled.
+                                          Default: 0
 
-error exit codes:
-  254: nothing to generate
-  255: general MPS error
+  --disable-strict-mode                   Disable strict generation mode. Strict mode places
```

Comment:
Let's call it `--no-strict-mode` for consistency with other command-line utilities.

Reply:
Renamed